README.md

```diff
@@ -76,6 +62,17 @@ Create a profile, review the sample use cases to familiarize yourself with the c
 
 You can now use all the functionalities of the extension.
 
+### Log in to API Mediation Layer and leverage existing base profile with token
```

Comment:
I am not sure if it should be mentioned...but if the host & port on the service & base profile are different, it will assume you are trying to use optional credentials, instead of the token. Ie. you will be prompted instead of the token working. Because of this line: https://github.com/zowe/vscode-extension-for-zowe/blob/5bb2157eb65411d54e800f6cf39351785e526802/src/Profiles.ts#L1147

Reply:
Very interesting. Should we be comparing them without doing a proper lookup?
What I'm trying to say is that `"example.com" !== "93.184.216.34"` even though they are the same.
Maybe we can target this comparison behavior in a future release. : )

But I think it's accurate that someone can login to APIML and use ZE with a token 😋

Reply:
CLI is not doing this and we have to do this in ZE because of optional cred profiles